frontend/public/components/graphs/area.tsx

```diff
@@ -121,6 +127,7 @@ type AreaChartProps = {
   tickCount?: number;
   title?: string;
   data?: DataPoint[];
```

Comment:
how about changing this to
```suggestion
  data?: DataPoint[][];
```

and then having
```javascript
<ChartGroup>
  {data.map((datum) => (
    <ChartArea data={datum} style={style} />
   ))}
</ChartGroup>
```
in render

Reply:
I would also use the same approach in UtilizationItem - `data` would be something like

```
data: {
  type: string //In, Out..
  data: DataPoint[]
}[]
```